style.css

```diff
@@ -2338,45 +2339,43 @@ object {
 
 .gallery-item {
 	display: inline-block;
-	text-align: center;
+	text-align: left;
 	vertical-align: top;
 	width: 100%;
+	margin: 0 0 1.5em;
+	padding: 0 1em 0 0;
+	width: 50%;
 }
 
-.gallery-columns-2 .gallery-item {
-	max-width: 50%;
-}
-
-.gallery-columns-3 .gallery-item {
-	max-width: 33.33%;
-}
-
-.gallery-columns-4 .gallery-item {
-	max-width: 25%;
-}
-
-.gallery-columns-5 .gallery-item {
-	max-width: 20%;
+.gallery-item a {
+	display: inline-block;
+	border-bottom: none;
+	background: none;
 }
 
-.gallery-columns-6 .gallery-item {
-	max-width: 16.66%;
+.gallery-item a:hover {
+	background: none;
+	
```

Comment:
There's an extra space here - the `}` should just be on the line immediately following `background: none;`.